test/Dockerfile.api

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:latest
+
+ENV WORK_DIR="/cyberark"
+RUN mkdir $WORK_DIR
+WORKDIR $WORK_DIR
```

Comment:
Nit: We can just use:
```
WORKDIR /cyberark
```
And strike lines 3 and 4. `WORKDIR` will automatically create it for us.